### PR TITLE
[Compute AG] Moved Talos Linux to Host OS section

### PR DESCRIPTION
**Pull request by Pubs-MV**
and fixed note

### CODE REVIEW

File: compute/admin_guide/install/system_requirements.adoc [hunk 1]
```diff
@@ -185,6 +185,21 @@ Runtime prevent capability is supported only for DNS events. Other prevent capab
...
 SLES 15  SP1 - SP4 - Only host Defenders are supported.
 
+|Talos OS
+|Talos OS 1.3.0
+Kernel version: 5.15.83-talos
+Kubernetes version: 1.25.4
+containerd version: 1.6.12
+
+
+The following features are not available for Talos OS.
```
Comment (grevach) [inline compute/admin_guide/install/system_requirements.adoc]: @Pubs-MV do we want to set this as a note? similar to bottlerocket

Comment (Pubs-MV) [reply compute/admin_guide/install/system_requirements.adoc]: done, and also clarifying that we do not support both Vuln and Compliance block policies

File: compute/admin_guide/install/system_requirements.adoc [hunk 2]
```diff
@@ -185,6 +188,26 @@ Runtime prevent capability is supported only for DNS events. Other prevent capab
...
+====
+The following features are not available for Talos OS:
+
+* Scanning of underlying hosts
+* Runtime scanning
+* Agentless scanning
+* Vulnerability and compliance blocking policies
+* Automatic recognition of the cluster name.
++
+You must enter it manually in the *Cluster name* field when deploying Defenders or twistcli
```
Comment (grevach) [inline compute/admin_guide/install/system_requirements.adoc]:
```suggestion
You can enter it manually in the *Cluster name* field when deploying Defenders via UI or twistcli
```

Comment (Pubs-MV) [reply compute/admin_guide/install/system_requirements.adoc]: done

**Pull request merged**